viewer/src/components/MoveButton.vue

```diff
@@ -205,13 +214,13 @@ export default class MoveButton extends Vue {
         this.startingHex = hex;
 
         const map: SpaceMap = this.$store.state.gaiaViewer.data.map;
-        const { range, costs } = this.button;
+        const {range, costs} = this.button;
```

Comment:
editor shenanigans

Reply:
I with the prettier would take care of it..